### ISSUE TITLE
Remove rancher-operator 

### ISSUE THREAD

**Issue by thedadams** [rancher/rancher#32179]
rancher-operator is no longer needed in Rancher v2.6+

**Comment by sowmyav27** [comment]
To test:
- Fresh install of master-head
- Upgrade from 2.5.8 to master-head

**Comment by sowmyav27** [comment]
@thedadams Can you let us know what resources make use of rancher-operator so that we can test an upgrade usecase and also the rollback from 2.6 to 2.5.8 (for instance) works?

**Comment by sangeethah** [comment]
from @cbron on workflows that uses rancher-operator:
Cluster is successfully registered in fleet once cluster is provisioned.
Clusters is assigned to the right fleet workspace(by default it is fleet-default), local cluster should be in fleet-local.
Cluster label from rancher gets populated into fleet cluster.

**Comment by anupama2501** [comment]
Verified on master-head commit d60543a
Fresh install
1. Once the cluster is created, from cluster explorer --> continuous delivery --> clusters --> fleet-default has the clusters registered but not the local cluster
2. cluster explorer --> continuous delivery --> clusters --> fleet-local has the local cluster and the default cluster group populated in the cluster groups
<img width="500" alt="Screen Shot 2021-05-24 at 10 22 23 AM" src="https://user-images.githubusercontent.com/38144301/119384718-fefd2600-bc79-11eb-9dbc-38e5b3c68baf.png">

**Comment by anupama2501** [comment]
@thedadams I have verified the above mentioned scenarios on a fresh install. can you let me know if I need to verify any other test cases?

**Comment by anupama2501** [comment]
Re-opening the issue:
On an upgrade from rancher version v2.5.8 to master-head commit d60543a:
rancher-operator is still seen in the local cluster workloads.

from cluster explorer --> local --> workloads --> rancher_operator should be removed after upgrading from v2.5.8
<img width="945" alt="Screen Shot 2021-05-24 at 11 21 01 AM" src="https://user-images.githubusercontent.com/38144301/119393149-6ec4de00-bc85-11eb-92b2-2196f7d391b9.png">

**Comment by anupama2501** [comment]
On v2.5.8 --> Created 2 rke1 node driver clusters from ember UI

- Verified rancher-operator is present in the local --> workloads

Upgraded to master-head d44698d:
- Verified the `rancher-operator` is removed from the workloads 

But the rke1 clusters are in `Modified` state with the following error in conditions:
```
Modified(1) [Bundle fleet-agent-c-nz79h]; clusterrole.rbac.authorization.k8s.io cattle-fleet-system-fleet-agent-role missing; clusterrole.rbac.authorization.k8s.io fleet-system-fleet-agent-role extra; deployment.apps cattle-fleet-system/fleet-agent missing; networkpolicy.networking.k8s.io cattle-fleet-system/default-allow-all missing```

**Comment by thedadams** [comment]
I am not able to reproduce the issue. I deployed Rancher, created a downstream cluster, upgraded Rancher, and the (1) rancher-opeator was removed and (2) the cluster in fleet renamed active.

**Comment by anupama2501** [comment]
Waiting on the issue #33221 to be resolved to verify the upgrade testcase

**Comment by anupama2501** [comment]
Verified upgrading from v2.5.8 to master-head ad099ee

On v2.5.8 verified rancher-operator is present in the local --> system project workload

After upgrade:

- `The rancher-operator is removed from the workloads.
- From Continuous Delivery --> clusters [namespace: fleet-default] shows all the clusters created before upgrade as `active`
- `fleet-local` namespace has the local cluster registered.
- Verified adding custom labels to the clusters is reflected in the continuous delivery --> fleet clusters
